test/test_block_store_service.py

```diff
@@ -171,8 +171,10 @@ def test_volume_properties(self):
                 self.assertEqual(test_vol.attachments.volume, test_vol)
                 self.assertEqual(test_vol.attachments.instance_id,
                                  test_instance.id)
-                self.assertEqual(test_vol.attachments.device,
-                                 "/dev/sda2")
+                # Device name mapping not available in azure
```

Comment:
Was there a resolution to this issue? I think it would be good to have a clear TODO: marker in the comments for this so we don't miss it. Also, it would be good to know what the MS cloud architects suggest for this.